packages/css/src/components/mega-menu/mega-menu.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-mega-menu {
+  padding-block: var(--amsterdam-mega-menu-padding-block);
+}
+
+.amsterdam-mega-menu__list-wrapper {
+  column-gap: var(--amsterdam-mega-menu-list-wrapper-column-gap);
+  columns: var(--amsterdam-mega-menu-list-wrapper-column-width, 20rem) auto;
+  padding-block-start: var(--amsterdam-mega-menu-list-wrapper-padding-block-start);
+
+  &:has(+ .amsterdam-heading) {
+    padding-block-end: var(--amsterdam-mega-menu-list-wrapper-padding-block-end);
+  }
```

Comment:
Does this work? You can't check previous siblings with `has`, right?

Reply:
This works, it checks if the list is followed by a heading and then applies some padding to the bottom.

Reply:
Pretty nifty, but a bit indirect.

This should be a grid or flex gap in the future, either by wrapping the second and third sections in a `Column` or applying a row span to the first.

For now, we can set the value of the `--amsterdam-mega-menu-list-wrapper-padding-block-end` token to the grid gap to reach the same visual effect. Probably separate for spacious and compact?

Reply:
This is for when two lists live in one grid cell, see the multiple categories variant in stories.